Content.Shared/Forensics/ForensicsComponent.cs

```diff
@@ -1,18 +1,23 @@
-namespace Content.Server.Forensics
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Forensics
 {
-    [RegisterComponent]
+    [RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
```

Comment:
This exposes the ForensicsComponent to all clients which is probably really bad in a server-authoritative context.

Reply:
Yeah but i need to read those data anyway.

If client read it, yes its annoying but if they cheat to get info they will be caught by admins after a moment.